one_fm/patches.txt

```diff
@@ -2,4 +2,6 @@
 one_fm.patches.v1_0.change_title_in_grd #2021-07-11
 one_fm.patches.v1_0.delete_fields #2021-07-11
 one_fm.patches.v1_0.assign_roles #2021-09-13
-one_fm.patches.v1_0.update_notification_logs #2021-12-15
\ No newline at end of file
+one_fm.patches.v1_0.update_notification_logs #2021-12-15
+one_fm.patches.v1_0.delete_interview_doctype
```

Comment:
Add dates as ID to these patches so they get executed exactly once.

Reply:
Patch will be executed only once

Reply:
Alright